application/controllers/admin/solutions.php

```diff
@@ -1,4 +1,17 @@
 <?php
+/**
+ *  Probably should not be here, but I don't know where to properly load the classes
+ */
+include_once "application/services/Formula/Node/Formula_node.php";
+include_once "application/services/Formula/Node/Formula.php";
+foreach (glob("application/services/Formula/Node/*.php") as $filename)
+{
+    include_once $filename;
+}
+include_once "application/services/Formula/NodeFactory.php";

```

Comment:
```suggestion
```

cely tento blok od riadku 2 az po riadok 11 tu nema co hladat, tie classy maju mat spravny PSR4 format pre autoloading a maju byt pridane do kodu pomocou `use`, tak ako to vidite na riadku 13 a 14